add azure region to email template (#6804)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/secure/databricks-privatelink.md

    - Databricks instance name:
    - Databricks Azure resource ID:
    - dbt Cloud multi-tenant environment: EMEA

>     - Azure region: Region where your Databricks workspace is hosted (for example, WestEurope, NorthEurope)

Author: ```suggestion
    - Azure region: Region that hosts your Databricks workspace (like, WestEurope, NorthEurope)
```
